src/translations/en.json

```diff
@@ -4396,9 +4396,9 @@
               "self_consumed_solar_could_not_calc": "Self-consumed solar energy couldn't be calculated"
             },
             "self_sufficiency_gauge": {
-              "card_indicates_self_sufficiency_quota": "This card indicates how self-sufficient your home is.",
-              "self_sufficiency_quota": "Self-sufficiency quota",
-              "self_sufficiency_could_not_calc": "Self-sufficiency quota couldn't be calculated"
+              "card_indicates_self_sufficiency_quota": "This card indicates how much of the energy consumed by your home came from on-site sources (such as solar panels or batteries).",
```

Comment:
I think "consumed energy" is too vague. But your suggestion makes me realize that there is inconsistency here with the other tooltips. I think that for clarity and consistency with the other tooltips, this could be either "the energy consumed by your home," or "the energy you consumed." If you pick one (I have no preference), I can do a PR later to migrate this over to the other tooltips.

Reply:
Thank you for pointing that out. I did not check the context. Given the choice, I'd pick _energy consumed by your home_.

Reply:
correct. let me remove my comment.